specification/network/resource-manager/Microsoft.Network/stable/2021-03-01/examples/PrivateEndpointCreate.json

```diff
@@ -24,7 +24,17 @@
         ],
         "subnet": {
           "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworks/myVnet/subnets/mySubnet"
-        }
+        },
+        "ipConfigurations": [
+          {
+            "name": "pestaticconfig",
+            "properties": {
+              "groupId": "file",
+              "memberName": "file",
+              "privateIPAddress": "192.168.0.5"
```

Comment:
Why would this IP be different than the response? This is 0.5 and the response is 0.6